transform/binary-plugin/binary_plugin.go

```diff
@@ -20,7 +20,7 @@ func NewBinaryPlugin(path string) transform.Plugin {
 	return &BinaryPlugin{commandRunner: &binaryRunner{pluginPath: path}, log: logrus.New().WithField("pluginPath", path)}
```

Comment:
This should create a PluginMetadata struct and set metadata.Name to `filepath.Base(path)`